dev-tools/install.sh

```diff
@@ -10,29 +25,31 @@ VERSION_CB="2.0.0.2"
 VERSION_SC="stable"
 
 echo λλλ phive
-if [ ! -f bin/phive ]; then
-    wget --no-clobber --output-document=bin/phive https://phar.io/releases/phive.phar
-    wget --no-clobber --output-document=bin/phive.asc https://phar.io/releases/phive.phar.asc
+if [ ! -x bin/phive ]; then
+    wget --output-document=- https://phar.io/releases/phive.phar > bin/phive
```

Comment:
What's the difference between `--output-document=bin/phive` and `--output-document=- > bin/phive`? Isn't removing `--no-clobber` enough?

Reply:
@julienfalque It won't overwrite the existing file. Just removing --no-clobber would start to number the file(s).

Reply:
Are you sure about that? I just tried this locally:
```console
$ wget --quiet --output-document=composer.json https://raw.githubusercontent.com/FriendsOfPHP/PHP-CS-Fixer/2.15/composer.json
$ head -n 2 composer.json
{
    "name": "friendsofphp/php-cs-fixer",
$ echo test > composer.json
$ head -n 2 composer.json
test
$ wget --quiet --output-document=composer.json https://raw.githubusercontent.com/FriendsOfPHP/PHP-CS-Fixer/2.15/composer.json
$ head -n 2 composer.json
{
    "name": "friendsofphp/php-cs-fixer",
```

Reply:
No, not sure. I normally refer to the manpage when I'm in doubt. `wget -O - ... > file` is the equivalent of `wget -O file ...` as I just read there, so what you suggested should behave the same as in the script (when not using `--no-clobber`).

Reply:
Actually, even if the previous syntax doesn't overwrite the existing file, I think it's fine because it means the file has been downloaded in a previous attempt, so it is already the expected one.

So I propose to:
- keep the previous syntax (no output redirect) as it looks more readable to me (removing the `--no-clobber` option should be fine)
- keep the `wget` execution in a `if [ ! -f file ]` condition as was done before (no need to try to download the file if it is already here) but keeping the `chmod u+x` execution in its own `if [ ! -x file ]` condition
- execute `gpg --recv-keys` command *before* downloading the file: if fetching the key fails but the file has been downloaded, it won't be tried again

Reply:
I can modify this so it pleases your preference regarding wget syntax.

But I would restrain from adding more complexity than necessary as I find that rather cumbersome for the other parts.

also I don't understand where you see the re-try transaction broken by the change-set for phive/gpg (as long as bin/phive is not executable, there is retry).

Reply:
The third point of my suggestion is necessary only if other items are applied. The current setup works fine. We might download the file even if we already did it successfully but it won't happen often so indeed, not worth the complexity.

Reply:
@julienfalque Just pushed an update. Rebased on current 2.15 as some travis build fixes are in there and added a new fixup! on top which removes relying on output redirection w/ wget and --output-file. I used -O as equivalent option name (short option) as it was in use in the script already to align w/ existing style. This should address your comments, so please take a look if this fits well in your eyes.

Reply:
Okay, retrying works however sometimes errors are of more permanent nature:

~~~
ERROR]    Failed to download load https://api.github.com/repos/maglnet/ComposerRequireChecker/releases: HTTP Code 403 
~~~

* Build: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/595557561

As I can't re-trigger the build AFAIK, I might push some faux commit later today to trigger it again.

Reply:
@ktomk `git commit --allow-empty -m 'Trigger CI'`